ADAL/src/public/ADTokenCacheItem.h

```diff
@@ -65,18 +67,18 @@
 @property (copy) NSString* familyId;
 
 /*! The access token received. Should be nil, in case the item stores multi-resource refresh token. */
-@property (copy) NSString* accessToken;
+@property (copy, nullable) NSString* accessToken;
 
 @property (copy) NSString* accessTokenType;
 
-@property (copy) NSString* refreshToken;
+@property (copy, nullable) NSString* refreshToken;
 
 @property (copy) NSData* sessionKey;
```

Comment:
sessionKey is also nullable, and usually is null. in fact the only thing in here that absolutely can't be null is authority.